src/SFA.DAS.EAS.Portal.Client/Database/Models/Document.cs

```diff
@@ -12,7 +12,7 @@ public abstract class Document : IDocument, IDocumentProperties, IOutbox
         public Guid Id { get; protected set; }
 
         [JsonIgnore]
-        public bool IsNew { get; protected set; }
+        public bool IsNew { get; set; }
```

Comment:
This is an internal property and shouldn't be able to be set from outside the object.  Please put back to protected.

Reply:
I agree it shouldn't be public, it was a (pained 😉) tactical decision to make it public, so that we could have a working unit test, without extending the time taken to complete the story. If `Document` was part of the public API, I'd have addressed it before declaring the PR ready to review, but I didn't, as it's only used internally in the Portal (the client only returns the contained `Account` object).

There are also other properties in `Document` which look like they should have more restrictive access modifiers. However, to make them more restrictive, we'd have to either refactor the code so that the test code doesn't need to set the property, or introduce a separate mechanism to allow the tests to modify it. (I know ideally tests shouldn't need to modify non-public properties!)

We could declare it as `protected internal` and make internals visible to the unit test project.

I can raise a tech debt item to address the access modifiers, unless you think that we need to address it as part of this PR?

Reply:
A unit test shouldn't have to set it.

Reply:
`IsNew` has been changed back to having a protected set. The unit test now creates the `AccountDocument` by deserializing it from json, which has the added bonus of more accuratly mirroring how its created in the test scenario.